test/integration/common/StudioActions.kt

```diff
@@ -115,16 +77,36 @@ object Utils {
         }
     }
 
-    fun cloneAndOpenProject(composeRule: ComposeContentTestRule, source: String, destination: String): Path {
-        val absolute = File(File(destination).absolutePath)
+    fun clickIcon(composeRule: ComposeContentTestRule, icon: Icon.Code) {
+        clickText(composeRule, icon.unicode)
+    }
+
+    fun clickText(composeRule: ComposeContentTestRule, text: String) {
+        composeRule.onNodeWithText(text).performClick()
+    }
 
-        absolute.deleteRecursively()
-        File(source).copyRecursively(overwrite = true, target = absolute)
+    fun nodeWithTextExists(composeRule: ComposeContentTestRule, text: String): SemanticsNodeInteraction {
+        return composeRule.onNodeWithText(text).assertExists()
+    }
 
-        StudioState.project.tryOpenProject(absolute.toPath())
+    fun nodeWithTextDoesNotExist(composeRule: ComposeContentTestRule, text: String) {
+        return composeRule.onNodeWithText(text).assertDoesNotExist()
+    }
```

Comment:
These functions are pretty thin, but we're doing this for the sake of an easier to read, more BDD-like test.